operators/src/plot/histogram.rs

```diff
@@ -393,6 +429,12 @@ struct HistogramMetadata {
     pub max: f64,
 }
 
+impl HistogramMetadata {
+    fn has_invalid_parameters(&self) -> bool {
+        self.number_of_buckets == 0 || self.min > self.max
```

Comment:
you need to check for min >= max i think

Reply:
359ff6d